news/2025/2025-11-11-resurrection-cup-2025-results.md

```diff
@@ -0,0 +1,456 @@
+---
+layout: post
+title: Resurrection Cup 2025 Results
+date: 2025-11-11 18:00:00 +0000
+---
+
+Intense matches, jaw-dropping highlights, and a vibrant production team working behind the scenes; this year's Resurrection Cup has proven to be one of the most exciting yet!
+
+![](/wiki/Tournaments/RESC/2025/img/banner.jpg)
+
+Let's dive into the results, highlights, and everything that made Resurrection Cup 2025: Regulus Revolution unforgettable.
+
+## Results
```

Comment:
the results section has no text and the highlights don't start with any leading paragraphs. feels very jarring when reading through when there's nothing to tie together the intro and the specific match highlights